src/ui/components/Badge/styles.scss

```diff
@@ -10,16 +10,25 @@
   display: inline-block;
   font-size: $font-size-s;
   line-height: 1.3;
+  margin-top: 10px;
   padding: 8px 12px 6px;
   text-align: center;
+
+  .Icon {
+    @include margin-end(10px);
+
+    vertical-align: middle;
+  }
 }
 
 .Badge-featured {
   background-color: $teal-50;
+}
 
-  .Icon-featured {
-    @include margin-end(10px);
+.Badge-restart-required {
+  background-color: $yellow-50;
 
-    vertical-align: middle;
+  .Icon-restart-required {
+    margin-top: -2px;
```

Comment:
With margin:

![screen shot 2017-08-24 at 17 54 26](https://user-images.githubusercontent.com/217628/29675388-4a7a3792-88f5-11e7-8ec5-27845f21c1a2.png)

Without:

![screen shot 2017-08-24 at 17 54 22](https://user-images.githubusercontent.com/217628/29675389-4a7b7986-88f5-11e7-9dfb-97e2d12f3365.png)

What I can do is use a different `vertical-align` though. What's the best?